src/main/java/io/vertx/core/loadbalancing/LoadBalancer.java

```diff
@@ -64,6 +69,67 @@ public <E> Endpoint<E> endpointOf(E endpoint) {
     }
   };
 
+  /**
+   * Random load balancer.
+   */
+  // LoadBalancer RANDOM = () -> (EndpointSelector) endpoints -> new Random().nextInt(endpoints.size());
+  // ThreadLocalRandom? contention, seed?
```

Comment:
Do you think ThreadLocalRandom should be used?

Reply:
I think we don't have any guarantee about the thread invoking the selector, so it would be safer to use `ThreadLocalRandom`